.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+on: [push]
+
+jobs:
+  test_job:
+    runs-on: ubuntu-latest
```

Comment:
It should not make a big difference, but we tend to use debian rather than ubuntu.

Reply:
I get "Can't find any online and idle self-hosted runner in the current repository, account/organization that matches the required labels: 'debian-latest'" when I try that.

It seems like GitHub has decided to stick with Ubuntu for all of their Linux CI hosts: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners

So, unfortunately, we will be limited to the Docker container for Debian.

Reply:
[SourceHut](https://sourcehut.org) offers [Debian VM images](https://man.sr.ht/builds.sr.ht/compatibility.md) and I'd be glad to integrate it with this repo if it is desired.

Reply:
Agreed @johanvdw , but @McSinyx if you have a plan how to do that integration, I would be fine with it.

Reply:
Doing it would require either trusting me for owner privilege or creating a paying account for FOSDEM ($20/year; builds.sr.ht got hit by cryptocurrency miners and decided to require payment earlier than initially planned).  Personally I'm OK with either but it's up for the maintainers to decide if it's worth it.